source/policy.rst

```diff
@@ -3,14 +3,14 @@
 Policy
 ======
 
-.. note:: This policy is of Oct 2016 and should be seen as a **first draft**. 
+.. note:: This policy is of Oct 2016 and should be seen as a **first draft**.
 
 Experiments require Firefox 50 or greater to work.
 
 Anyone can write a WebExtension Experiment, there is no need to ask for permission from anyone to do so.
 
 Currently Experiments can only be run on Firefox Nightly and Firefox Developer Edition. This is to prevent APIs that are generally seen to be experimental being run on release versions of Firefox.
 
-Currently to upload an Experiment to AMO_ and get it signed you must be a member of certain group of users at Mozilla.
+Currently to upload an Experiment to AMO_ and get it signed you must be a member of certain group of users at Mozilla. If you are not a member of that group then `users can turn off the requirement for signing <https://wiki.mozilla.org/Add-ons/Extension_Signing#FAQ>`_ on Firefox Nightly
 
```

Comment:
Is that the only option?  What if we want to accept a contribution from community?  Who do they ask to upload the experiment?